tailwind.config.cjs

```diff
@@ -2,7 +2,8 @@
 module.exports = {
   content: ["./index.html", "./src/**/*.{vue,js}"],
   theme: {
-    extend: {},
+    extend: {
```

Comment:
Can we revert the changes in this file? Thanks!

Reply:
sure!